releasenotes/notes/annotation-context-modify-name-and-prompt-cc74b3b268983181.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    LLM Observability: Introduces `prompt` and `name` arguments to ``LLMObs.annotation_context`` to support setting an integration generated span's name and `prompt` field.
```

Comment:
Let's add a link to the public corp docs for annotation context here

Reply:
Just going to link to the "annotating a span" section since the exact path for annotation contexts is still TBD